lib/DDG/Spice/GetEvents/Nearby.pm

```diff
@@ -1,15 +1,15 @@
-package DDG::Spice::GetEvents;
+package DDG::Spice::GetEvents::Nearby;
```

Comment:
Hi @moollaza.
Following your advise I'm trying to unify the two IA and I'm having a few difficulties.
The basic issue is that the parameters to the api in the two case are different. In one case I have to call 'https://api.getevents.co/ddg?city=$1' and on the other the call is 'https://api.getevents.co/ddg?lat=$1&lng=$2&timezone=$3'. I tries two approaches:
(a) Change the 'spice to' inside the handle subroutine.
(b) Use spice to => 'https://api.getevents.co/ddg?$1' and return a $1 value which includes both the parameter names and their values.

Neither approach worked.
Further, I'm not sure I fully understand your last response:
spice is_cached => 1; # by default, cache nginx rewrites
spice proxy_cache_valid => "200 12h"; #cache successful API responses for 12h
...
if (is_local) {
    return $result, {is_cached => 0}; # don't cache nginx rewrite, caches API response
} else {
        return $result; # caches nginx rewrite, caches API response
}
-What is $result? What should be it's value?
-What is is_local? Is that just a boolean variable that tells in which case are we?